modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,176 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, OpenShift Service Mesh (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like OpenShift Monitoring for user-defined projects.
```

Comment:
Could you make "OpenShift Monitoring for user-defined projects" a link to this page https://docs.openshift.com/container-platform/4.13/monitoring/enabling-monitoring-for-user-defined-projects.html?

Reply:
Linking directly in the body isn't possible as it breaks other systems that use this content, so I added an "Additional Resources" section and linked to "Enabling Monitoring for User-Defined Projects" from there.